tests/cli/helpers/event_filters.py

```diff
@@ -62,7 +96,11 @@ def testAddArguments(self):
     event_filters.EventFiltersArgumentsHelper.AddArguments(argument_parser)
 
     output = self._RunArgparseFormatHelp(argument_parser)
-    self.assertEqual(output, self._EXPECTED_OUTPUT)
+
+    if tuple(sys.version_info[0:2]) < (3, 6):
```

Comment:
Please move this to py2to3 instead

Is this really specific to 3.6? Doesn't appear on 3.5?

Reply:
Tested with `tox -epy35` also surfaces on 3.5 I'll make the changes and add an issue to add tests for 3.5

Reply:
```
----------------------------------------------------------------------
Ran 1249 tests in 267.843s

OK (skipped=7)
_____________________________________________________________________________________________________ summary _____________________________________________________________________________________________________
  py35: commands succeeded
  congratulations :)
```